src/cript/api/api.py

```diff
@@ -235,7 +232,7 @@ def __str__(self) -> str:
         ...     storage_token=os.getenv("CRIPT_STORAGE_TOKEN")
         ... ) as api:
         ...     print(api)
-        CRIPT API Client - Host URL: 'https://api.criptapp.org/api/v1'
+        CRIPT API Client - Host URL: 'https://api.criptapp.org'
```

Comment:
moving my comment from the other PR:

i'm curious is this change cascading to other aspects of the code (implications for me) or no ? I will just leave comments here and there to try to thouroughly understand as much as possible

Reply:
It should not have no effect (or a manageable effect) on the cascade downstream.

Since most API requests are capsuled after this, the capsuled request autmotically appends this for you.
(It can be optionally disabled.)
And that should handle %95 of all situations.

If it is not handled by this, you can recover the old behavior by

`hostname = f"{api.host}/{api.api_prefix}/{api.api_version}"`